msteams-platform/resources/dev-preview/developer-preview-tools.md

```diff
@@ -1,27 +1,39 @@
 ms.date: 08/10/2018
 ---
-# DevTools for the Microsoft Teams Desktop Client
+# DevTools for Microsoft Teams tabs
 
 When Teams is running in a browser, it’s easy to access the browser's DevTools: F12 (on Windows) or Command-Option-I (on MacOS).
 
-While the web version of Teams and the desktop version of teams are almost exactly the same, there are some differences, particularly with respect to authentication. Sometimes the only way to figure out what’s going on is to use the DevTools. Here's how to get to them from the Teams desktop client.
+The feature is only available in desktop and Android clients after Developer Preview has been enabled. See [How do I enable Developer Preview](~/resources/dev-preview/developer-preview-intro.md) for more information.
 
-The feature is only available in the desktop Developer Preview version of Microsoft Teams. See [How do I get access to the Developer Preview](~/resources/dev-preview/developer-preview) for more information. These tools are not available for the mobile client.
+## Accessing DevTools in the Desktop
 
-## Accessing DevTools in the Teams Client
+While the web version of Teams and the desktop version of teams are almost exactly the same, there are some differences, particularly with respect to authentication. Sometimes the only way to figure out what’s going on is to use the DevTools. Here's how to get to them from the Teams desktop client. To use DevTools in the desktop client:
 
-To use DevTools in the desktop client, first open up a tab so you have something to inspect with the DevTools, then open DevTools:
-
-* On Windows, you open DevTools via the Microsoft Teams icon in the desktop tray:
+1. Make sure you have enabled [developer dreview](~/resources/dev-preview/developer-preview-intro.md)
+1. Open up a tab so you have something to inspect with the DevTools.
+1. Open the DevTools
+    * On Windows, you open DevTools via the Microsoft Teams icon in the desktop tray:
 
   ![Right-click to open DevTools](~/assets/images/dev-preview/devtools-right-click.png)
 
-* On MacOS, you open DevTools by clicking on the Microsoft Teams icon in the Dock.
+    * On MacOS, click on the Microsoft Teams icon in the Dock.
 
 Here’s what a sample tab looks like with the DevTools open and an element selected:
 
 ![Tab and DevTools](~/assets/images/dev-preview/tab-and-devtools.png)
+
+## Accessing DevTools from an Android client
+
+You can also enable the DevTools from the Teams Android client. To do so:
+
+1. Make sure you have enabled [developer dreview](~/resources/dev-preview/developer-preview-intro.md)
+1. Connect your device to your desktop computer, and setup your Android device for [remote debugging](https://developers.google.com/web/tools/chrome-devtools/remote-debugging/)
+1. In your Chrome browser, open `chrome://inspect/#devices`.
+1. Click **inspect** below the tab you wish to debug, as in the screenshot below.
+
+![Android DevTools](~/assets/images/android-devtools.png)
```

Comment:
Want to add one more section here:

Below mentioned debugging capabilities will be available on Android client:
1.	View console logs. 
2.	View/modify html, css, network requests runtime. 
3.	Put breakpoints in javascript and debug the code.

Reply:
Isn't that what developer tools means? Calling that functionality out specifically for Android seems odd. I added this to the opening section instead.